File: Teltonika.App/Models/TrackingVehicleViewModel.cs
using System;

namespace Teltonika.App.Models
{
    public class TrackingVehicleViewModel
    {
        public short Altitude { get; set; }
        public short Direction { get; set; }
        public double Lat { get; set; }
        public double Long { get; set; }
        public byte Priority { get; set; }
        public byte Satellite { get; set; }
        public double Speed { get; set; }
        public DateTime DateTimeUtc { get; set; }
        public string Status { get; set; }
        public bool StopFlag { get; set; }
        public bool IsStop { get; set; }
        public string Alarm { get; set; }
        public int Mileage { get; set; }
        public int Temprature { get; set; }
        public string Address { get; set; }
        public string Imei { get; set; }

    }
}

File: Teltonika.App/ClientApp/src/models/TrackingVehicleViewModel.ts
export class TrackingVehicleViewModel {
    constructor(
        public Altitude: number,
        public Direction: number,
        public Lat: number,
        public Long: number,
        public  Speed : number,
        public Timestamp: Date,
        public  Status : string,
        public  StopFlag : boolean,
        public IsStop: boolean,
        public  Alarm :string,
        public  Mileage : number,
        public  Temprature : number,
        public Address: string,
        public Imei : string
    ) { }
    

}